ccd-definition/CaseEventToFields/WorkAllocationIntegrationFields-SDO.json

```diff
@@ -116,5 +129,15 @@
     "PageID": "WorkAllocationIntegrationFields",
     "PageDisplayOrder": 11,
     "Publish": "Y"
+  },
+  {
```

Comment:
Remember to move this field to WorkAllocationIntegrationFields-SDO-CUI-nonprod.json